app/models/UserAnswers.scala

```diff
@@ -24,7 +24,7 @@ import scala.util.{Failure, Success, Try}
 
 trait ReadableUserAnswers {
   val data: JsObject
-  val is5mldEnabled: Boolean = false
+  val is5mldEnabled: Boolean = true
```

Comment:
I would consider leaving this as `false` while 5mld is not enabled in production